### ISSUE TITLE
.gitignore /vendor

### ISSUE THREAD

**Issue by kliqks** [deliciousbrains/wp-migrate-db-anonymization#17]
I don't know when it happened, but it appears that /vendor is added to the gitignore. At least in my deployment, this makes the entire vendor folder not added to the commit and therefore, I imagine the whole plugin will fail to work. As a matter of fact, the vendor folder appears to be missing from this repo as well.

**Comment by polevaultweb** [comment]
@kliqks I don't commit the vendor directory here, but package it with the build that does to the WordPress repo. I clarified that here with install instructions: https://github.com/deliciousbrains/wp-migrate-db-anonymization#installation

**Comment by kliqks** [comment]
That's cool, but the gitignore with the /vendor line will keep people that use the plugin from the WordPress plugin store from committing that folder in their own repo's. Seriously, I might be the only one doing it this way, but it's how I need to do it for my AWS High Availability sites. For git purposes, maybe this needs to be included in your .git/exclude file instead of using the gitignore. Thank you though, I love this plugin.

**Comment by polevaultweb** [comment]
@kliqks good point, but that is an edge case. I'll look at removing the .gitignore for the build. Any reason you aren't using Composer to manage the plugins, that way they stay out of your git repos?
https://deliciousbrains.com/using-composer-manage-wordpress-themes-plugins/
